code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -61,7 +61,7 @@
 	return 1
 
 
-/obj/effect/proc_holder/changeling/sting/transformation
+obj/effect/proc_holder/changeling/sting/transformation
```

Comment:
This isn't a string through, it's a path, and all absolute paths should start with a `/`. Functionally there's no difference, but the lack of the `/` *technically* makes it relative pathing, which should be avoided because it needlessly makes grep/searching much more difficult.